Generate Certificates guide (#2212)
sensu/sensu-docs: content/sensu-go/5.18/guides/securing-sensu.md

- [Secure the API and dashboard](#secure-the-api-and-dashboard)
- [Secure Sensu agent-to-server communication](#secure-sensu-agent-to-server-communication)
- [Sensu agent TLS authentication](#sensu-agent-tls-authentication)
- [Next step: Run a Sensu cluster](#next-step-run-a-sensu-cluster)

As with any piece of software, it is critical to minimize any attack surface the software exposes.
Sensu is no different.
This guide describes the components you need to secure to make Sensu production-ready.

> To use this guide, you must have previously [generated the certificates][1] you will need to secure Sensu in production.

Reviewer: `that you will need for securing Sensu in production` ?

The sentence itself sounds off to me.

Reviewer 2: ```suggestion
Before you can use this guide, you must have [generated the certificates][1] you will need to secure Sensu.
```

Reviewer 2: @Blue0ctober WDYT? ☝️

Reviewer: LGTM

You can use use certificates for authentication that are distinct from other communication channels used by Sensu, like etcd or the API.
However, deployments can also use the same certificates and keys for etcd peer and client communication, the HTTP API, and agent authentication without issues.

## Next step: Run a Sensu cluster

Well done!

> Your Sensu installation should now be secured with TLS.

Author: Need to make sure this secure Sensu guide actually does configure TLS by the end of the guide. We may want to make this clearer with more direction.
